sbt-plugin/src/main/scala/akka/grpc/sbt/AkkaGrpcPlugin.scala

```diff
@@ -58,6 +58,7 @@ object AkkaGrpcPlugin extends AutoPlugin {
       case object Java extends Language
     }
 
+    val akkaGrpcProtocOptions = settingKey[Seq[String]]("Additional options to be passed to protoc")
```

Comment:
I don't think we should add this setting: users can just add things to `PB.protocOptions` directly.

Reply:
As long as we can give an example somewhere.

Reply:
Yes, definitely! For example in one of the 'scripted' tests, and perhaps also linking a snippet from the docs?